pkg/image/image_test.go

```diff
@@ -150,3 +152,154 @@ func TestReader(t *testing.T) {
 		img.File.Close()
 	}
 }
+
+func TestAuthorizedPath(t *testing.T) {
+	tests := []struct {
+		name       string
+		path       []string
+		shouldPass bool
+	}{
+		{"empty path", []string{""}, false},
+		{"invalid path", []string{"/a/random/invalid/path"}, false},
+		{"valid path", []string{"/"}, true},
+	}
+
+	// Create a temporary image
+	path := downloadImage(t)
+	defer os.Remove(path)
+
+	// Now load the image which will be used next for a bunch of tests
+	img, err := Init(path, true)
+	if err != nil {
+		t.Fatal("impossible to load image for testing")
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			auth, err := img.AuthorizedPath(test.path)
+			if test.shouldPass == false && (auth == true && err == nil) {
+				t.Fatal("invalid path was reported as authorized")
+			}
+			if test.shouldPass == true && (auth == false || err != nil) {
+				if err != nil {
+					t.Fatalf("valid path was reported as not authorized: %s", err)
+				} else {
+					t.Fatal("valid path was reported as not authorized")
+				}
+			}
+		})
+	}
+}
+
+func TestAuthorizedOwner(t *testing.T) {
+	type ownerGroup struct {
+		name       string
+		owners     []string
+		shouldPass bool
+	}
+
+	// Note that we do not test the "root" case because the result will depend
+	// on the context of the test execution and therefore sometimes pass,
```

Comment:
Calling `test.EnsurePrivilege(t)` at the beginning of this function should fix that

Reply:
fixed